mods/ra2/rules/allied-infantry.yaml

```diff
@@ -125,26 +125,64 @@ e1:
 		HP: 125
 	Mobile:
 		Speed: 31
+		RequiresCondition: !chronodisable && undeployed
 	Passenger:
 		PipType: Green
 	RevealsShroud:
 		Range: 5c0
-	AttackFrontal:
-		Voice: Attack
 	Armament@primary:
 		Weapon: M60
-		RequiresCondition: !rank-elite
+		RequiresCondition: !rank-elite && !deployed
 	Armament@elite:
 		Weapon: M60E
-		RequiresCondition: rank-elite
-	# Armament@deployed:
-	#	Weapon: para
-	#	UpgradeTypes: deployed
-	#	UpgradeMinEnabledLevel: 1
-	# Armament@elite-deployed:
-	#	Weapon: paraE
-	#	UpgradeTypes: eliteparaweapon
-	#	UpgradeMinEnabledLevel: 1
+		RequiresCondition: rank-elite && !deployed
+	Armament@deployed:
+		Weapon: para
+		Name: deployed
+		RequiresCondition: !rank-elite && deployed
+	Armament@elite-deployed:
+		Weapon: paraE
+		Name: deployed
+		RequiresCondition: rank-elite && deployed
+	GrantConditionOnDeploy:
+		DeployedCondition: deployed
+		UndeployedCondition: undeployed
+		CanDeployOnRamps: True
+		Facing: 160
```

Comment:
Is there any way for us to get the facing dynamically?

Us forcing it to this value means the sprite comes out of the undeploy sequence and may abruptly face a different direction.

Not a block but could be polished (potentially in the future with upstream changes).

Reply:
I think this should be 128 (probably leftover from SP Marauder, which is where i copied the code from).

What is required to make the turret realign after undeploy, but not while deployed, so after deploy the turret facing won't jump. There should be an issue to make turret realign before deploy on upstream.

There is only one anim so not forcing a facing (not defining this) doesn't really help.

Reply:
Let's link the ticket here if we can and change this to 128 then if you believe that value is more correct.